src/SlackMessage.js

```diff
@@ -79,26 +79,23 @@ class SlackMessage extends SlackMessageRoot {
       linkObj = { url: link, text: link };
     }
 
-    const response = await this.#_slackGateway.fetchMessage(
+    const message = await this.#_slackGateway.fetchMessage(
       this.#_channel,
       this.#_ts
     );
 
-    if (response.body.ok) {
-      const message = response.body.messages[0];
```

Comment:
I didn't like that this code had to know to grab the first message out of the response body, so I moved that little bit of logic to the SlackGateway.